keps/sig-auth/20190806-serviceaccount-tokens.md

```diff
@@ -263,6 +264,35 @@ service account token on behalf of pods running on that node. The
 NodeRestriction admission controller will require that these tokens are pod
 bound.
 
+### Safe Adoption
+
+#### Blocker: In-cluster clients should reload fresh token
+
+- Summary: Bound service account tokens are time-bound. If in-cluster clients 
+do not load fresh token from projected volume, tokens would be rejected since 
+they are expired.
+
+- Rollout plan:
+  1. Modify admission-injected token source to request for token expiring at an 
+  uncommon value. (1 hour 10 seconds for example)
+  1. Modify kube apiserver TokenRequest API, providing an option 
+  `--mint-longer-token` to mint a token longer than requested if expiration 
```

Comment:
nit: suggest something with a `--service-account-` prefix to sit next to the other flags, e.g. `--service-account-extend-injected-expiration`

Reply:
proposed --service-account-extend-token-expiration